standalone/test/integration_tests/api/integrationtest_scanner_api.cpp

```diff
@@ -163,21 +162,40 @@ void ScannerAPITests::setUpScannerHwMock()
   hw_mock_.reset(new StrictMock<ScannerMock>{ HOST_IP_ADDRESS, port_holder_ });
 }
 
-TEST_F(ScannerAPITests, shouldSendStartRequestAndReturnValidFutureWhenLaunchingWithValidConfig)
+class ScannerAPITestsFragmented : public ScannerAPITests
+{
+  void SetUp() override
+  {
+    ScannerAPITests::SetUp();
+    setUpScannerConfig();
+    setUpScannerV2Driver();
+    setUpScannerHwMock();
+  }
+};
+
+class ScannerAPITestsUnfragmented : public ScannerAPITests
+{
+  void SetUp() override
+  {
+    ScannerAPITests::SetUp();
+    setUpScannerConfig(HOST_IP_ADDRESS, UNFRAGMENTED_SCAN);
+    setUpScannerV2Driver();
+    setUpScannerHwMock();
+  }
+};
+
+TEST_F(ScannerAPITestsFragmented, shouldSendStartRequestAndReturnValidFutureWhenLaunchingWithValidConfig)
```

Comment:
For this test (and probably a few others) the fragmented property is not relevant.

I suggest to use the `ScannerAPITests` fixture for these test cases and also setup a default config in `ScannerAPITests.SetUp()`. It will get overwritten in the child classes anyways.

Reply:
Eventually, we can decide if it makes sense to sort the tests according to their fixture. Or even separate the integrationtest into multiple files with an own header for the common fixture.

Reply:
either the fragmented or the unfragmented property has to be the defautl case. up to now it was the fragmented one. switching to ScannerApiTests even with adding the default config there, would resault in having to create driver and mock objects again. Or if you add those creations to the ScannerAPITests.SetUp, you will have problems with "shouldThrowWhenConstructedWithInvalidLaserScanCallback" which does set up the scanner itself and uses no hw_mock.